pkg/identity/authorize.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Can you add some unit tests for these?

Reply:
Done!

Reply:
Did you have tests for extractIssuer?

Reply:
replaced extractissuer with extractIssuerURL which already has tests and is exactly the same